src/sass/rx/_partials/_rx-prescriptions.scss

```diff
@@ -39,36 +40,44 @@
 
   a {
     text-decoration: none;
+    
+    &:hover {
+      background: transparent;
+      border-bottom: 2px solid currentColor;
+    }
   }
 }
 
+.rx-prescription-refilled {
+  margin-bottom: 5rem;
+}
+
 .rx-prescription-number {
   font-weight: bold;
 }
 
-.rx-prescription-refilled {
-  padding-bottom: 3rem;
+.rx-prescription-button {
+  white-space: nowrap;
 }
 
 button.rx-prescription-button,
 [type=submit].rx-prescription-button {
   background: transparent;
   display: inline-block;
   font-weight: bold;
-  float: right;
   margin: 0;
 
   @media (max-width: $small-screen) {
-    max-width: 60%;
+    // max-width: 60%;
     padding: 1rem;
   };
 }
 
 .rx-prescription-count {
-  display: inline-block;
   position: absolute;
-  bottom: 1.5rem;
-  
+  bottom: 0;
+  display: inline-block;
+  line-height: 1; 
```

Comment:
Seems to line up the bottom edges of the action and count better if you put `line-height: 2rem` on `.rx-prescription-countaction` instead.